tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -845,6 +845,7 @@ def cleanup_gaia(self, full_reset=True):
 
         if full_reset:
             # disable passcode
+            # XXX should I even change this?! ---v
```

Comment:
This is about setting up the defaults. If you feel it's safe to do so, we can just remove this line.